modules/oadp-using-enable-api-group-versions.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: PROCEDURE
+[id="oadp-using-enable-api-group-versions_{context}"]
+= Working with different Kubernetes API versions in the same OADP operation
+
+By default, Velero only backs up resources that use the preferred version of the Kubernetes API. However, Velero also includes a feature, link:https://velero.io/docs/v1.9/enable-api-group-versions-feature/[Enable API Group Versions], that overcomes this limitation. When enabled on the source cluster, this feature causes Velero to back up _all_ Kubernetes API group versions that are supported on the cluster, not only the preferred one. Once the versions are stored in the backup tarball file, they are available to be restored on the destination cluster.
+
+For example, a source cluster with an API named `Example` might be available in the `example.com/v1` and `example.com/v1beta2` API groups, with `example.com/v1` being the preferred API.
+
+Without the Enable API Group Versions enabled, Velero backs up only the preferred API group version for `Example`, which is `example.com/v1`. With the feature enabled, Velero also backs up `example.com/v1beta2`.
+
+When the Enable API Group Versions feature is enabled on the destination cluster, Velero selects the version to restore on the basis of the order of priority of API group versions.
+
+[NOTE]
+====
+Enable API Group Versions is still in beta.
+====
+
+Velero uses the following algorithm to assign priorities to API versions, with 1 as the top priority:
+
+. Preferred version of the _destination_ cluster+
+. Preferred version of the source_ cluster  +
+. Common non-preferred supported version with the highest Kubernetes version priority
+
+.Procedure
+
+* Configure the `EnableAPIGroupVersions` feature flag:
+
+[source, yaml]
+```
+apiVersion: oadp.openshift.io/vialpha1
+kind: DataProtectionApplication
+...
```

Comment:
See this comment https://github.com/openshift/openshift-docs/pull/48202/files#r957990388 for my questions about this API.